### PR TITLE
Finished mobile menu including its transistion. Was told we will like…

### PR DESCRIPTION
**Pull request by HereItGoes**
…ly be using icon so currently using word as placeholder once icon is added it should align with the expand button

## Change Summary
**[Briefly summarise the changes that you made. Just high-level stuff]**

### Change Form
**Fill this up (NA if not available). If a certain criteria is not met, can you please give a reason.**

- [ ] The pull request title has an issue number
- [ ] The change works by "Smoke testing" or quick testing
- [ ] The change has tests
- [ ] The change has documentation

## Other Information
**[Is there anything in particular in the review that I should be aware of?]**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -31,7 +31,8 @@
     "next": "11.0.1",
     "react": "17.0.2",
     "react-dom": "17.0.2",
-    "react-hook-form": "7.9.0"
+    "react-hook-form": "7.9.0",
+    "yarn": "^1.22.10"
```
Comment (JeremiahPinto) [inline package.json:35]: Why would you install yarn?

File: components/MobileMenu/MobileMenu.tsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
+
+import React from 'react'
```
Comment (JeremiahPinto) [inline components/MobileMenu/MobileMenu.tsx:2]: Don't need to import react

File: styles/main.css [hunk 3]
```diff
@@ -67,3 +67,40 @@
...
+  margin-top: 10%;
+  padding: 5%;
+  background-color: #51185f;
+}
+
+@media only screen and (min-width: 767px) {
+  .menu {
+    display: none;
+  }
+}
```
Comment (JeremiahPinto) [inline styles/main.css:106]: All of this can be written using tailwind classes

File: components/MobileMenu/MobileMenu.tsx [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+
+import { Popover,Transition } from '@headlessui/react'
+
+function MobileMenu() {
```
Comment (JeremiahPinto) [inline components/MobileMenu/MobileMenu.tsx:6]: Change to es6 function declaration

**Review by JeremiahPinto — CHANGES_REQUESTED** [review]
Unnecessary nesting of component name for the mobile menu. Change it to `index.tsx`

File: components/MobileMenu/MobileMenu.tsx [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+        leave="transition duration-400 ease"
+        leaveFrom="transform scale-100 opacity-100"
+        leaveTo="transform scale-95 opacity-0">
+          <Popover.Panel className="menu-list">
+            <div className="menu-ele">
+              <a href="/">Home</a>
+              <a href="/">Superhero</a>
+              <a href="/">Recipe</a>
+              <a href="/">Progress</a>
+              <a href="/">Sign In</a>
```
Comment (JeremiahPinto) [inline components/MobileMenu/MobileMenu.tsx:22]: Fix these link issues, they need to follow next.js link format

**Comment by HereItGoes** [comment]
There were issue with my laptop for some reason I'd I didnt do those it
wouldn't work.

File: package.json (hunk 1 above)
Comment (HereItGoes) [reply package.json:35]: For some reason when I run it on my laptop it wasnt working unless i install it there were defect in that laptop it should run after removing those

File: components/MobileMenu/MobileMenu.tsx (hunk 2 above)
Comment (HereItGoes) [reply components/MobileMenu/MobileMenu.tsx:2]: For some reason when I run it on my laptop it wasnt working saying react not found unless i put it there were defect in that laptop it should still run after removing those

**Pull request closed without merging**